packages/seo-store/src/analysis/tests.js

```diff
@@ -26,5 +27,97 @@ describe( "Config slice", () => {
 				analysisType: "term",
 			} );
 		} );
+
+		test( "should update whether the SEO analysis is active", () => {
+			const { updateIsSeoActive } = configActions;
+
+			const result = configReducer( previousState, updateIsSeoActive( false ) );
+
+			expect( result ).toEqual( {
+				...initialState,
+				isSeoActive: false,
+			} );
+		} );
+
+		test( "should update whether the readability analysis is active", () => {
+			const { updateIsReadabilityActive } = configActions;
+
+			const result = configReducer( previousState, updateIsReadabilityActive( false ) );
+
+			expect( result ).toEqual( {
+				...initialState,
+				isReadabilityActive: false,
+			} );
+		} );
+
+		test( "should add a research", () => {
+			const { addResearch } = configActions;
+
+			const result = configReducer( previousState, addResearch( "test" ) );
+
+			expect( result ).toEqual( {
+				...initialState,
+				researches: [ "morphology", "test" ],
+			} );
+		} );
+
+		test( "should remove a research", () => {
+			const { removeResearch } = configActions;
+
+			const result = configReducer( previousState, removeResearch( "morphology" ) );
+
+			expect( result ).toEqual( {
+				...initialState,
+				researches: [],
+			} );
+		} );
+	} );
+
+	describe( "Selectors", () => {
+		const createStoreState = curry( set )( {}, "analysis.config" );
+
+		test( "should select the configuration", () => {
+			const { selectAnalysisConfig } = configSelectors;
+
+			const result = selectAnalysisConfig( createStoreState( initialState ) );
+
+			expect( result ).toEqual( initialState );
+		} );
+
+		test( "should select the analysis type", () => {
+			const { selectAnalysisType } = configSelectors;
+
+			const state = { ...initialState, analysisType: "category" };
+			const result = selectAnalysisType( createStoreState( state ) );
+
+			expect( result ).toBe( "category" );
+		} );
+
+		test( "should select the is SEO active", () => {
+			const { isSeoAnalysisActive } = configSelectors;
+
+			const state = { ...initialState, isSeoActive: false };
+			const result = isSeoAnalysisActive( createStoreState( state ) );
+
+			expect( result ).toBe( false );
+		} );
+
+		test( "should select the is readability active", () => {
+			const { isReadabilityAnalysisActive } = configSelectors;
+
+			const state = { ...initialState, isReadabilityActive: false };
+			const result = isReadabilityAnalysisActive( createStoreState( state ) );
+
+			expect( result ).toBe( false );
+		} );
+
+		test( "should select the researches", () => {
+			const { selectResearches } = configSelectors;
+
+			const state = { ...initialState, researches: [ "morphology", "test" ] };
```

Comment:
```suggestion
			const state = { analysisType: "category" };
			const result = selectAnalysisType( createStoreState( state ) );

			expect( result ).toBe( "category" );
		} );

		test( "should select the is SEO active", () => {
			const { isSeoAnalysisActive } = configSelectors;

			const state = { isSeoActive: false };
			const result = isSeoAnalysisActive( createStoreState( state ) );

			expect( result ).toBe( false );
		} );

		test( "should select the is readability active", () => {
			const { isReadabilityAnalysisActive } = configSelectors;

			const state = { isReadabilityActive: false };
			const result = isReadabilityAnalysisActive( createStoreState( state ) );

			expect( result ).toBe( false );
		} );

		test( "should select the researches", () => {
			const { selectResearches } = configSelectors;

			const state = { researches: [ "morphology", "test" ] };
```

Suggestion to remove the `initialState` as it isn't being used in these tests.

Reply:
I like tests to simulate the real usage. But you are right that these selectors only use those values. So I guess I'm fine with either 😉